server/routes/activities/allocation-dashboard/handlers/allocationDashboard.ts

```diff
@@ -77,7 +77,7 @@ export default class AllocationDashboardRoutes {
         slots[slot.weekNumber][`timeSlots${day}`] ??= []
 
         if (slot[`${dayLowerCase}Flag`]) {
-          slots[slot.weekNumber].days.push(dayLowerCase)
+          if (!slots[slot.weekNumber].days.includes(dayLowerCase)) slots[slot.weekNumber].days.push(dayLowerCase)
```

Comment:
This is unrelated to the fix, but I noticed duplicate days can be added to this array.